manifest.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name"				:	"Duo Strength",
 	"description"		:	"Adds individual skill strengths back into the duolingo webpage, similar to data on duome.eu",
-	"version"			:	"1.1.6",
+	"version"			:	"1.1.7",

```

Comment:
As said previously, the target branch is now develop, so the version number doesn't need to be changed. This can be done once the changes end up in the release branch.